src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1528,4 +1528,80 @@ private class ACCT_IndividualAccounts_TEST {
         System.assertEquals('Ms. Betty, Lily and Tom White Family', requeryFinalAccount.Name);
 
     }
+
+    /*********************************************************************************************************
+    * @description In a multi-currency enabled org, create a contact with different currency than the default one.
+    * Make sure that the account automatically created and the contact have the same currency.
+    */
+    @isTest
+    private static void insertContactMultiCurrencyEnabled() {
```

Comment:
I may have missed it but do we want unit tests for Admin Accounts as well?

Reply:
Good catch. Maybe we can remove that setting setup line so that it is using default model?

Reply:
do you mean line 1549?